src/test/groovy/io/github/cosmicsilence/scalafix/ScalafixPropsTest.groovy

```diff
@@ -107,4 +107,12 @@ class ScalafixPropsTest extends Specification {
         '2.12.10'           || "org.scalameta:semanticdb-scalac_${projectScalaVersion}:${ScalafixProps.scalametaVersion}"
         '2.13.0'            || "org.scalameta:semanticdb-scalac_${projectScalaVersion}:${ScalafixProps.scalametaVersion}"
     }
+
+    def 'it should return the semanticdb artifact coordinates for #scalametaVersion meta version'() {
```

Comment:
sug.: `'it should return the semanticdb artifact coordinates using a version override'`?